aws-rds-dbinstance/aws-rds-dbinstance.json

```diff
@@ -0,0 +1,431 @@
+{
```

Comment:
Could you elaborate why `EnableCustomerOwnedIp , NcharCharacterSetName , TdeCredentialArn` parameters are not included in the schema? 
reference: https://docs.aws.amazon.com/AmazonRDS/latest/APIReference/API_CreateDBInstance.html

Reply:
None of these are present in the current spec: https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-rds-database-instance.html
We can take a note to add support of these in the future as a separate PR.